persistence/src/main/resources/reference.conf

```diff
@@ -213,3 +213,10 @@ akka.actor {
   }
 }
 
+# Cluster distribution settings
+akka.cluster.distribution {
```

Comment:
I'd recommend against using the `akka` namespace for things that don't live in Akka